specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -88,6 +88,149 @@
         ]
       },
       "x-ms-parameter-location": "method"
+    },
+    "WeatherYear": {
+      "name": "year",
+      "description": "Year of the cyclone(s)",
+      "type": "integer",
+      "format": "int32",
+      "required": true,
+      "in": "query",
+      "x-ms-parameter-location": "method"
+    },
+    "RequiredWeatherBasinID": {
+      "name": "basinID",
+      "type": "string",
+      "description": "Basin identifier",
+      "required": true,
+      "in": "query",
+      "enum": [
+        "AL",
+        "EP",
+        "SI",
+        "NI",
+        "CP",
+        "NP",
+        "SP"
+      ],
+      "x-ms-enum": {
+        "name": "basinID",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "AL",
+            "description": "AL"
+          },
+          {
+            "value": "EP",
+            "description": "EP"
+          },
+          {
+            "value": "SI",
+            "description": "SI"
+          },
+          {
+            "value": "NI",
+            "description": "NI"
+          },
+          {
+            "value": "CP",
+            "description": "CP"
+          },
+          {
+            "value": "NP",
+            "description": "NP"
+          },
+          {
+            "value": "SP",
+            "description": "SP"
+          }
+        ]
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "OptionalWeatherBasinID": {
+      "name": "basinID",
```

Comment:
Let's follow existing parameter naming convention to be camel case, basinId.

Reply:
Updated